src/main/java/org/ohdsi/circe/cohortdefinition/CorelatedCriteria.java

```diff
@@ -37,6 +37,6 @@
   @JsonProperty("Occurrence")
   public Occurrence occurrence;
   
-  @JsonProperty("restrictVisit")
+  @JsonProperty("RestrictVisit")
```

Comment:
Thank you for changing this to follow the other case conventions of the criteria objects.  While this doesn't follow java naming conventions, it is what was established, and we would have difficulty changing all the other properties to be camelCase due to existing data that is out three. So this is a good fix to find now before it becomes widely available.